docs/UserGuide.md

```diff
@@ -40,95 +50,163 @@ Format: `help`
 
+Format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
+
+Examples: 
+* `add water the plants, 0800, high, daily`
+* `add facial appointment, jul 10, med, monthly`
+
+#### Listing all tasks : `list`
+Shows a list of all tasks in a particular day.<br>
+Format: `list DATE`
+
+#### Searching tasks by keywords: `search`
+Searches and lists all tasks that contain the specified keyword.<br>
+Format: `search KEYWORD`<br>
+
+> * The search is not case sensitive.
+> * The order of the keywords does not matter. e.g. `buy milk oct 21` will match `oct 21 buy milk`
+> * Only full words will be matched e.g. `buy milk` will not match `buy cow`
+> * Tasks matching keywords will be returned
+
+Examples: 
+* `search oct 1`<br>
+  Lists all tasks that have the date `oct 1`.
+* `search milk`<br>
+  Lists the all the tasks with names containing the word `milk` if any can be found.
+  
 #### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Deletes the specified task from the list. Irreversible.<br>
+Format: `delete INDEX...`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
+* `delete 2`<br>
+  Deletes the task indexed number second in the current seen list.
+* `search buy milk`<br> 
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the task indexed first in the results of the `search` command.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+To delete more than one task, simply add in the other indexes separated with a comma. <br>
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Example:<br>
+* `delete 1, 2, 3`
+* `search oct 1`
+  `done 2, 3`
+  
+  
+#### Undo : `undo`
+Reverts the most recent action.<br>
+Format: `undo`
+
+#### Editing a task : `edit`
+Edits task on the displayed list.<br>
+Format: `edit INDEX PARAMETER NEW_INFORMATION`
+
+<img src="images/2ndimpt.png" width="500"><br>
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `edit 1 date oct 11`
+* `list dec 30`<br>
+  `edit 5 task buy coconut`
+  
+To make one of the parameters (except the task name) empty, follow the format below:<br>
+Format: `edit INDEX PARAMETER clear`
+
+Example: 
+* `edit 3 time clear`
+
+
+#### Sorting all tasks : `sort`
```

Comment:
in the context of a to-do manager, is sort a necessity? if you think it is, then retain it. it is also OK to remove this feature.